### PR TITLE
MINOR: test case for usage->delete->usage

### PR DESCRIPTION
**Pull request by sushi30**
### Describe your changes:

Add [test case](https://github.com/open-metadata/OpenMetadata/blob/4f5e7ecc0408e24000014c6dbb88d775d73e0159/ingestion/tests/integration/postgres/test_postgres.py#L252) to demonstrate https://github.com/open-metadata/OpenMetadata/issues/16408.

This test runs:
1. ingest
2. usage
3. delete service
4. ingest
5. usage

The second ingestion usage fails due to the DB being corrupted with queries pointing to tables that have been deleted. The only way to recover the DB is to delete the queries which point to non-existent tables.

#
### Type of change:

- [x] Bug fix

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: bootstrap/sql/migrations/native/1.4.1/mysql/postDataMigrationSQLScript.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+-- matchEnum Test Definition Parameter for columnValuesToBeInSet 
+UPDATE test_definition
+set json = JSON_MERGE_PRESERVE(
```
Comment (pmbrull) [inline bootstrap/sql/migrations/native/1.4.1/mysql/postDataMigrationSQLScript.sql:3]: hi sorry silly question, this does not belong to the use case defined in the PR right

File: ingestion/tests/integration/postgres/test_postgres.py [hunk 2]
```diff
@@ -204,3 +208,96 @@ def test_profiler(run_profiler_workflow):
...
+    }
+    workflow = UsageWorkflow.create(workflow_config)
+    search_cache.clear()
+    workflow.execute()
+    workflow.raise_from_status()
+
+
+@pytest.mark.xfail(
+    reason="'metadata.ingestion.lineage.sql_lineage.search_cache' gets corrupted with invalid data."
+    " See issue https://github.com/open-metadata/OpenMetadata/issues/16408"
```
Comment (pmbrull) [inline ingestion/tests/integration/postgres/test_postgres.py:250]: ah got it, we just want this to fail for now and fix it later

File: openmetadata-docs/content/v1.3.x/main-concepts/metadata-standard/schemas/tests/column/columnValueMaxToBeBetween.md [hunk 3]
```diff
@@ -1,16 +0,0 @@
...
-
-*This schema defines the test ColumnValueMaxToBeBetween. Test the maximum value in a col is within a range.*
-
-## Properties
-
-- **`minValueForMaxInCol`** *(integer)*: Expected maximum value in the column to be greater or equal than.
-- **`maxValueForMaxInCol`** *(integer)*: Expected maximum value in the column to be lower or equal than.
-
-
-Documentation file automatically generated at 2022-07-14 10:51:34.749986.
```
Comment (pmbrull) [inline openmetadata-docs/content/v1.3.x/main-concepts/metadata-standard/schemas/tests/column/columnValueMaxToBeBetween.md:16]: where did we move these docs?

File: ingestion/tests/integration/postgres/test_postgres.py (hunk 2 above)
Comment (sushi30) [reply ingestion/tests/integration/postgres/test_postgres.py:250]: yes. this way we can deliver a test before delivering the fix.

File: bootstrap/sql/migrations/native/1.4.1/mysql/postDataMigrationSQLScript.sql (hunk 1 above)
Comment (sushi30) [reply bootstrap/sql/migrations/native/1.4.1/mysql/postDataMigrationSQLScript.sql:3]: yes.. I branched off from a feature branch.. after the merge the diff should be cleaned

File: openmetadata-docs/content/v1.3.x/main-concepts/metadata-standard/schemas/tests/column/columnValueMaxToBeBetween.md (hunk 3 above)
Comment (sushi30) [reply openmetadata-docs/content/v1.3.x/main-concepts/metadata-standard/schemas/tests/column/columnValueMaxToBeBetween.md]: this diff is not clean.. in any case they are here:
https://github.com/open-metadata/OpenMetadata/blob/a4c516d2c70c69ae8cd27096da077a5b62d4a53f/openmetadata-docs/content/v1.3.x/connectors/ingestion/workflows/data-quality/tests.md#L4

**Comment by sushi30** [comment]
@pmbrull diff is clean now. feel free to take another look

**Review by pmbrull — APPROVED** [review]

**Pull request merged**